### DISCUSSION TITLE
Does Fast DDS and RPC have support for android system?

### DISCUSSION THREAD

**Discussion by ChandanGirish** [eProsima/Fast-DDS#3949] [Unofficial support]
I am trying to built android application with DDS RPC, Is there any support to cross compile Fats DDS RPC to android system?

**Comment by JLBuenoLopez** [comment]
@ChandanGirish 

There are no current official support for your use case. You might want to [contact eProsima's commercial support team](https://github.com/eProsima/Fast-DDS#getting-help) to discuss options on how to proceed.
